### ISSUE TITLE
Angular 7 and ng-library support

### ISSUE THREAD

**Issue by mahcr** [PatrickJS/starter#2049]
**Note: for support questions, please use one of these channels:** [Chat: AngularClass.slack](http://angularclass.com/member-join/) or [Twitter: @TipeIO](https://twitter.com/TipeIO)

* **I'm submitting a ...**
[ ] bug report
[x] feature request
[ ] question about the decisions made in the repository

* **Do you want to request a *feature* or report a *bug*?**
Angular 7 support and libraries

* **What is the current behavior?**
There is not support 

* **What is the motivation / use case for changing the behavior?**
The base is already outdated, avoid the angular-cli dependency because of the libraries, 

* **Please tell us about your environment:**

- Angular version: n/a
- Browser: n/a

* **Other information** (e.g. detailed explanation, stacktraces, related issues, suggestions how to fix, links for us to have context, eg. stackoverflow, gitter, etc)

**Comment by Uter1007** [comment]
I am already using Angular 7 already with this stack. Should work out of the box...

**Comment by mahcr** [comment]
@Uter1007 sure, I suppose that is just increase the number but there is not support for libraries

**Comment by BruneXX** [comment]
Hi guys, is there a due date for this? is this repository currently maintained? I saw a lot of pull request just there without attention.

**Comment by dvaldivia** [comment]
@Uter1007 did you had to adapt any of the code for it to work on angular 7? have you tried 8? I'm getting the types errors for `rxjs`
```
node_modules/rxjs/internal/types.d.ts(81,44): error TS1005: ';' expected.
```

**Comment by bbrink68** [comment]
Any update on this? I'm on Angular 6 and getting it. Using rxjs version `^6.5.3`.

**Comment by Druson** [comment]
I fixed this by changing `"typescript": "~2.7.2"` to `"typescript": "~2.8"`
